ksql-common/src/main/java/io/confluent/ksql/serde/Delimiter.java

```diff
@@ -35,15 +35,15 @@
 
   private final char delimiter;
 
-  public static Delimiter of(final char ch) {
+  public static Delimiter parse(final char ch) {
```

Comment:
As per last review, can we keep this one as `of` please? It's not doing any parsing...

Reply:
Uhh what a pain. What's happened here is the default IntelliJ behaviour of renaming all overloaded methods with same name has caught me. Will fix in another PR.